lib/active_model/serializer/railtie.rb

```diff
@@ -2,9 +2,7 @@
 module ActiveModel
   class Railtie < Rails::Railtie
     initializer 'active_model_serializers.logger' do
-      ActiveSupport.on_load(:action_controller) do
-        ActiveModelSerializers.logger = ActionController::Base.logger
-      end
+      ActiveSupport.on_load(:active_model_serializers) { self.logger ||= Rails.logger }
```

Comment:
This won't work for two reasons:

1) It's not our lib that needs to be loaded for Rails.logger to be defined, it's Rails.
2) This won't work because there's nothing running this hook